tests/apps/configurationapp/app.go

```diff
@@ -54,6 +54,11 @@ var (
 	grpcClient      runtimev1pb.DaprClient
 )
 
+type UnsubscribeConfigurationResponse struct {
+	Ok      bool   `protobuf:"varint,1,opt,name=ok,proto3" json:"ok,omitempty"`
+	Message string `protobuf:"bytes,2,opt,name=message,proto3" json:"message,omitempty"`
```

Comment:
@shivamkm07 why is this protobuf tag needed?

Reply:
Not required, removed